docs/index.md

```diff
@@ -880,6 +880,62 @@ Note that a `Triton-Datacenter-Name` response header was added in 9.2.0.
 
 The section describes API changes in CloudAPI versions.
 
+## 9.14.0
+
+- Expose Feed of Machines Changes [#68](https://github.com/joyent/sdc-cloudapi/pull/68).
+  Requires activation through SAPI:
+
+```
+cloudapi_svc=$(sdc-sapi /services?name=cloudapi|json -H 0.uuid)
+sapiadm update $cloudapi_svc metadata.experimental_cloudapi_changefeed=true
+```
+
+The following is an example of using `node-triton` to watch the changes:
+
+```
+$ ./bin/triton -i changefeed
+Change (2020-11-27T18:11:58.633Z) =>
+    modified: state,zone_state,tags
+    state: shutting_down
+    object: 98cbd3d0
+Change (2020-11-27T18:11:58.636Z) =>
+    modified: state,zone_state,pid,tags
+    state: down
+    object: 98cbd3d0
+Change (2020-11-27T18:11:58.641Z) =>
+    modified: state,zone_state,boot_timestamp,pid,tags
+    state: ready
+    object: 98cbd3d0
+Change (2020-11-27T18:12:00.632Z) =>
+    modified: state,zone_state,boot_timestamp,pid,tags
+    state: running
+    object: 98cbd3d0
+Change (2020-11-27T18:12:00.634Z) =>
+    modified: tags
+    state: ready
+    object: 98cbd3d0
+Change (2020-11-27T18:12:00.636Z) =>
+    modified: tags
+    state: running
+    object: 98cbd3d0
```

Comment:
I'm still trying to figure out what's the reason for VMAPI publishing these messages. For example, right now I've issued a `start` request for the same machine. In the middle of the VMAPI logs I have the following:

```[2020-11-30T11:55:05.407Z] DEBUG: vmapi/FastClientRequest/25578 on a09406e1-9c94-46fc-a844-2aafd99358e2: rpc putObject: entered (req_id=b2e5d778-2c32-403b-bbc7-ad079155a885, domain=moray.coal.joyent.us, msgid=5536, rpcmethod=putObject, timeoutms=null)
    rpcargs: [
      "vmapi_changefeed",
      "9625eefd-2577-48f5-b367-5ee805fbc845",
      {
        "changeKind": {
          "resource": "vm",
          "subResources": [
            "tags"
          ]
        },
        "changedResourceId": "98cbd3d0-5d6f-429d-8535-bf6ba5afe1e3",
...
```

Trying to dig further using `sdc-req` didn't point out anything else so, for some reason, VMAPI is re-writing machine records due to _tags_?. This clearly deserves an issue, but I'm trying to figure out the reason for this before I fill it. Any hint you may have regarding this would be very appreciated.

Speaking of CloudAPI's publishing these changes, I could just wipe out the messages including just tags, but that would break the listener when we're trying to watch to tags changes for real.

Reply:
After some more digging, this happens only when a machine hasn't got any tags assigned. There's something trying to save a machine w/o tags with `tags: {}` compared with another instance w/o tags.

For machines with tags, this doesn't happen:
```
Change (2020-11-30T17:26:35.642Z) =>
    modified: state,zone_state,pid
    state: down
    object: cde1f7a9
Change (2020-11-30T17:26:35.644Z) =>
    modified: state,zone_state
    state: shutting_down
    object: cde1f7a9
Change (2020-11-30T17:26:37.645Z) =>
    modified: state,zone_state,boot_timestamp,pid
    state: ready
    object: cde1f7a9
Change (2020-11-30T17:26:37.646Z) =>
    modified: state,zone_state,boot_timestamp,pid
    state: running
    object: cde1f7a9
```

Reply:
Ok, found it: while `_loadVm` function used at the beginning of VMAPI requests doesn't add any tags to the VM object if not already set, the update-vm workflow puts the whole thing, included `tags: {}` as expected for a VM w/o them to the request. Therefore the differences between both objects.

I'm not sure if `_loadVm` function should or not add `tags: {}` by default (it's what `getVm` does) or we should just double check that if the only difference for two VM objects are the tags, and these are equal to `{}` we should just move forward.

I'm inclined to say that we should be consistent and always set tags to an empty object if nothing is present. Thoughts?

Reply:
Good finding! Consistency would be good - so always setting tags to {} sounds good to me.